### PR TITLE
Fix: Add missing object text variant for Super Lotus and Super Hacker

### PR DESCRIPTION
**Pull request by xezon**
Add dedicated object text for Super Lotus and Super Hacker. Construction texts already say "Super Lotus" and "Super Hacker".

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/InfantryGeneral.ini [hunk 1]
```diff
@@ -1340,7 +1340,7 @@ Object Infa_ChinaInfantryHacker
   End
 
   ; ***DESIGN parameters ***
-  DisplayName      = OBJECT:Hacker
+  DisplayName      = OBJECT:SuperHacker ; Patch104p @tweak from OBJECT:Hacker
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/InfantryGeneral.ini]: Why is one a bugfix, the other a tweak?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/InfantryGeneral.ini]: Fixed

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 2]
```diff
@@ -20591,6 +20607,22 @@ RU: "Хакер"
 AR: "ﺮﺗﻮﯿﺒﻣﻮﻜﻟا ﺺﻟ"
 END
 
+// Patch104p @bugfix xezon 15/05/2023 Add specific object title for Super Hacker.
+
+OBJECT:SuperHacker
+US: "Super Hacker"
+DE: "Super-Hacker"
+FR: "Super technopirate"
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/generals.str:20615]: Still laughing at technopirate

**Review by commy2 — APPROVED** [review]

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**